docs-app/package.json

```diff
@@ -42,14 +42,14 @@
     "test:device": "ember test"
   },
   "devDependencies": {
-    "@babel/core": "^7.26.10",
+    "@babel/core": "^7.27.1",
     "@ember/optional-features": "^2.2.0",
     "@ember/string": "^4.0.1",
-    "@ember/test-helpers": "^5.2.0",
-    "@embroider/broccoli-side-watch": "^1.0.1",
-    "@embroider/compat": "^3.8.5",
-    "@embroider/core": "^3.5.5",
-    "@embroider/macros": "^1.16.12",
+    "@ember/test-helpers": "^5.2.1",
+    "@embroider/broccoli-side-watch": "^1.1.0",
+    "@embroider/compat": "^3.9.0",
+    "@embroider/core": "^3.5.6",
```

Comment:
At the moment, `@embroider/compat@4.x`, `@embroider/core@4.x`, and `@embroider/router@3.x` don't support apps with Webpack.